R/nm-join-bayes.R

```diff
@@ -480,7 +480,9 @@ sim_ipred <- function(mod, mod_mrgsolve, exts, data, join_col, y_col, pbar) {
         ipar_n <- dplyr::filter(ipar, .data$draw == ext_row$draw)
         mrgsolve::idata_set(mod_sim, ipar_n) %>%
           mrgsolve::smat(mrgsolve::as_bmat(ext_row, "SIGMA")) %>%
-          mrgsolve::mrgsim_df(obsonly = TRUE, carry_out = join_col) %>%
+          mrgsolve::mrgsim_df(
+            obsonly = TRUE, carry_out = join_col, etasrc = "data.all"
```

Comment:
Per discussion, this should be `etasrc = "idata.all"`

This also means we need to require a later `mrgsolve`